### PR TITLE
Fix intent resolving behavior for Android 11

### PR DESCRIPTION
**Pull request by warwicklol14**
Fixes : #3067 

Signed-off-by: Varun Paul kjvpaul@tutanota.com

### Pull Request Checklist

- [X] Changes has been tested on an Android device or Android emulator with API 21
- [ ] UI change has been tested on both light and dark themes
- [X] Pull request is based on the develop branch
- [X] Pull request updates [CHANGES.md](https://github.com/vector-im/element-android/blob/develop/CHANGES.md)
- [ ] Pull request includes screenshots or videos if containing UI changes
- [X] Pull request includes a [sign off](https://github.com/matrix-org/synapse/blob/master/CONTRIBUTING.md#sign-off)

### CODE REVIEW

File: vector/src/main/java/im/vector/app/features/widgets/WidgetFragment.kt [hunk 1]
```diff
@@ -236,11 +236,9 @@ class WidgetFragment @Inject constructor() :
                 val context = requireContext()
                 val intent = Intent.parseUri(url, Intent.URI_INTENT_SCHEME)
                 if (intent != null) {
-                    val packageManager: PackageManager = context.packageManager
-                    val info = packageManager.resolveActivity(intent, PackageManager.MATCH_DEFAULT_ONLY)
```
Comment (bmarty) [inline vector/src/main/java/im/vector/app/features/widgets/WidgetFragment.kt:240]: @ganfra can you tell me why we had this code please?

**Review by bmarty — APPROVED** [review]
LGTM, thanks for the update!

File: vector/src/main/java/im/vector/app/features/widgets/WidgetFragment.kt (hunk 1 above)
Comment (ganfra) [reply vector/src/main/java/im/vector/app/features/widgets/WidgetFragment.kt:240]: I guess it was cleaner than doing try/catch? 
For the issue see: https://stackoverflow.com/questions/62535856/intent-resolveactivity-returns-null-in-api-30

**Comment by warwicklol14** [comment]
No problem. Glad to help!

File: vector/src/main/java/im/vector/app/features/widgets/WidgetFragment.kt (hunk 1 above)
Comment (bmarty) [reply vector/src/main/java/im/vector/app/features/widgets/WidgetFragment.kt:240]: yes, it's cleaner than try catch, but I was wondering about the usage of `PackageManager.MATCH_DEFAULT_ONLY`

**Comment by bmarty** [comment]
I have added the `<queries>` tag in #3238, so this PR is not necessary anymore. Anyway, thanks again @warwicklol14 for your contribution.

**Pull request closed without merging**